### PR TITLE
Added delete workspace dao method, tested No Vertex Left Behind

### PR DESCRIPTION
**Pull request by helgridly**

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+/**
+ * Created with IntelliJ IDEA.
+ * User: hussein
+ * Date: 08/18/2015
+ * Time: 15:05
+ */
+class GraphDAODeleteSpec extends FreeSpec with Matchers with OrientDbTestFixture {
+  val wsName = WorkspaceName("myNamespace", "myWorkspace")
+  val workspace = Workspace(wsName.namespace, wsName.name, "aBucket", DateTime.now, "testUser", new HashMap[String, Attribute]() )
```
Comment (mattsooknah) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala]: Why a ```HashMap``` specifically?

Comment (helgridly) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala]: Because HashMaps are efficient. Especially empty ones.

Comment (mattsooknah) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala]: Is efficiency a concern here though?

Comment (mattsooknah) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala]: Also if there's a non-empty version of the workspace below, maybe ``wsEmpty``` would be a clearer name for this.

**Comment by jmthibault79** [comment]
:+1:

File: src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala (hunk 1 above)
Comment (helgridly) [inline src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAODeleteSpec.scala]: No. I'm being an ass.

**Pull request merged**